index.js

```diff
@@ -68,7 +68,11 @@ function formatWithCursor(text, opts, addAlignmentSize) {
     return { formatted: text };
   }
 
-  text = stripBom(text);
+  const UTF8BOM = 0xfeff;
+  const hasUnicodeBOM = text.charCodeAt(0) === UTF8BOM;
+  if (hasUnicodeBOM) {
+    text = stripBom(text);
```

Comment:
Should we do `.substring(1)` instead?

Reply:
Yeah, I was kinda thinking the same (or `.slice(1)` for consistency with other code). There's almost no reason to continue using `strip-bom` here, now that we have to keep track of what a BOM is in order to re-print it.

Reply:
@azz  done